src/config/hypre_macros_misc.m4

```diff
@@ -102,29 +102,29 @@ then
       gcc|mpigcc|mpicc)
         CFLAGS="-O2"
         if test "$hypre_using_openmp" = "yes" ; then
-          CFLAGS="$CFLAGS -fopenmp"
-          LDFLAGS="$LDFLAGS -fopenmp"
+          CFLAGS+=" -fopenmp"
```

Comment:
Why these changes?

Reply:
I think the new form is clearer and safer. Even though this example is simple, I think it's a good practice to use "+="

Reply:
Why is it safer?

Reply:
Mainly because it reduces error risk since you don't need to type CFLAGS, for example, twice.

Reply:
I don't see any harm using the '+=' since we have moved away from using the Bourne Shell and have now adopted much of what is offered in bash.  I see the point about potentially making mistakes by typing the variable name in twice.  I also see the potential for mistakes by forgetting the extra space in the '+=' approach.  The '+=' syntax is more readable IMO.  I'm happy either way here.

Reply:
Thanks! I'll consider this as resolved then :)